tools/network-scene-vis.md

```diff
@@ -0,0 +1,61 @@
+
+The visualizations are updated in real-time as the network messages are sent and received, showing the game state synchronization across the network.
+
+NetSceneVis can help diagnose issues related to network latency, packet loss, and other network-related issues that may affect the player experience in your multiplayer games. This tool can help optimize your network code to ensure that your game runs smoothly for all players.
+
+<!-- For full release: ## Requirements
+- Unity 2023.2
+:::note
+Unity 2023.2.0a11 is specifically required for NetSceneVis mesh shading
+:::
+- Up-to-date Multiplayer Tools package, see [Install the Multiplayer Tools Package](./install-tools.md). -->
+
+<!-- Update the following Requirements section after full release for this tool. -->
+## Requirements
+- Unity 2023.2 (This version of the editor is in pre-release. See [Unity 2023.1 Beta Resources](https://unity.com/releases/editor/beta) for more information.)
+:::note
+Unity 2023.2.0a11 is specifically required for NetSceneVis mesh shading
+:::
+- Up-to-date Multiplayer Tools package from the `develop` branch.
+
+### Installing Network Scene Visualization from `develop`
+
+You can install the `develop` branch of the Multiplayer Tools package with either a git URL or as a local git clone.
+
+### Installing from a git URL
+
+1. Open your Unity Hub and select the Project your are presently working on.
+2. In the editor's menu bar, navigate to **Window** > **Package Manager**.
+3. Click the plus sign ![plus sign](/img/add.png) in the Package Manager status bar and select **Add package from git URL...**.
+4. Enter `https://github.com/Unity-Technologies/com.unity.multiplayer.tools` in the pop-up window and select **Add**.
+5. After downloading, **Multiplayer Tools** appears under **Unity Technologies** in your **Package Manager** window.
+
+### Installing from a local git clone
+Instructions for installing a package from a local folder can be found here. A local clone of the Multiplayer Tools package can be created using the following git command:
+
+git clone https://github.com/Unity-Technologies/com.unity.multiplayer.tools.git
+
+1. Open your Unity Hub and select the Project your are presently working on.
+2. In the editor's menu bar, navigate to **Window** > **Package Manager**.
+3. Click the plus sign ![plus sign](/img/add.png) in the Package Manager status bar and select **Add package from disk...**.
+4. Navigate to the folder root of your local package.
+5. Double-click the `package.json` file in the file browser. The file browser closes, and the package now appears in the package list with the  tag.
+
+:::note
+If you have a package installed locally that matches a version in the registry, the **Update** button still appears and the list might display the same version twice as **Currently Installed** and **Recommended**.
+:::
```

Comment:
I think this is based on the internal instructions for internal users to install the package from git before it had been fully released. However, now that there's an official package release for it I don't think it should be necessary for users to install this package via git. End users may also not have the required permissions to install this package from git.

Reply:
Makes sense. Had a commented section to update once it was official the package went out. I'll make the update.

Reply:
@ian-m-unity Even in Unity 2023.2.0a13, the Multiplayer Tools package is only 1.1.0 and does not include NetSceneVis. Since NetSceneVis is part of 2.0.0 prerelease, I think it still needs to be manually added from `develop`.

Reply:
Installing by git URL is also not working to install 2.0.0pre

Reply:
Install by git clone works, but is that accessible to our users since the repo is private?

Reply:
Considering this resolved based on Slack convo. Updated the doc for installing tools package through the editor based on when it should hit the editor's package manager.